doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,37 +57,45 @@ Values are provided in kilobytes (KB).
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Release)  |            28 |               630 |              4 |         32 |         694 |                             161 |
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | Sample                                                             |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      +====================================================================+===============+===================+================+============+=============+=================================+
+      | :ref:`Light Bulb <matter_light_bulb_sample>` (Debug)               |            28 |               794 |              4 |         32 |         858 |                             185 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Bulb <matter_light_bulb_sample>` (Lto debug)           |            28 |               732 |              4 |         32 |         796 |                             189 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Bulb <matter_light_bulb_sample>` (Release)             |            28 |               691 |              4 |         32 |         755 |                             179 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Switch <matter_light_switch_sample>` (Debug)           |            28 |               757 |              4 |         32 |         821 |                             176 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Switch <matter_light_switch_sample>` (Lto debug)       |            28 |               693 |              4 |         32 |         757 |                             180 |
```

Comment:
```suggestion
      | :ref:`Light Switch <matter_light_switch_sample>` (LTO, Debug)      |            28 |               693 |              4 |         32 |         757 |                             180 |
```

Debug and Release are established terms, and LTO is just an additional build feature